style/runtimeTemplate.css

```diff
@@ -26,7 +26,20 @@
   margin-top: 10px;
   outline: none;
 }
-
+.create-runtime-radio {
+  display: flex;
+  align-items: center;
+  margin-left: -10px;
```

Comment:
We should not use negative values. Please check if there is a better way of handling this